README.md

```diff
@@ -114,9 +114,11 @@ If the data is seemingly out of sync with production you may want to get a more
 
 In order to update your local data with the most recent version of production you will need to have access to our production database, pg_dump, and access to the cms.gov vpn.
 
+If adding a new model to this process please refer to the adding a new model process below.
+
 1.  Connect to the VPN and run the following command below for postgresql in the command line.  Replace db_address and port_number with the database address and port number respectively.  A file called `backup.sql` should populate in the directory the command is run in.
 ```
-pg_dump -h <db_address> -p <port_number> -U eregsuser -f backup.sql -t 'search_sy*' -t 'resources_*' --data-only --column-inserts eregs
+pg_dump -h cmcs-eregs-site-prod-rdsresource-encrypted-cluster15-cluster.cluster-cicxftrvatqb.us-east-1.rds.amazonaws.com -p 3306 -U eregsuser -f backup.sql -t 'search_sy*' -t 'resources_*' -t 'regulations_statute*' --data-only --column-inserts eregs
```

Comment:
I rather not post what our host name is for security reasons. Its best to leave it as db_address

Reply:
also the port too. best to give minimum info.